### DISCUSSION TITLE
the millisecond part of the timestamp is displayed incorrectly

### DISCUSSION THREAD

**Discussion by Yurii-kyiv** [dbeaver/dbeaver#17881] [Q&A]
### Description

select now();
now                           
------------------------------
2022-10-10 11:**19:12**.**1912** +0300
millisecond part always repeats data from minutes and seconds
It is DBeaver's bug, not DB
Native Postgresql client app psql displays timestamp correctly

### DBeaver Version

Community Edition 22.1.3.202207240747

### Operating System

Windows 10

### Database and driver

PostgreSQL 10/11

### Steps to reproduce

Global Settings\Editors\Data Editor: Template for "Timestamp with Timezone" = "yyyy-MM-dd HH:mm:ss.**ms** Z"

**Comment by LonwoLonwo** [comment]
Hello @Yurii-kyiv 

As I can see in your pattern and result:
HH: (Hour in day) - 11
mm: (Minute in hour) - 19
ss: (Second in minute) - 12
m: (Minute in hour without leading zeros) - 19
s: (Second in minute without leading zeros) -12 

So, 11:19:12.1912 - is expected result for your pattern. And here you can find our wiki page: https://dbeaver.com/docs/wiki/Managing-Data-Formats/#time

Do you want to see millisecond in the result value? Then you can try to use default DBeaver timestamp pattern with SSS on the end.

![2022-10-11 02_41_42-Preferences](https://user-images.githubusercontent.com/45152336/194968265-e7638ece-0430-4d21-83a8-767369773738.png)

**Comment by Yurii-kyiv** [comment]
Thanks a lot Anastasiya.
I was wrong. I used Postgresql template for timestamp.

**Comment by LonwoLonwo** [comment]
DBeaver is based on java, and it is a universal tool, so we are using java patterns, yes.

Ok. Does this issue solve? Am I closing the ticket?

**Comment by Yurii-kyiv** [comment]
Yes, issue solved. Thanks
